metrics/network/remote_network/README.md

```diff
@@ -71,3 +73,15 @@ $ cd metrics/network/remote_network
 $ bash remote-networking-cpu.sh -i "<interface_name>" -u "<user>" -a "<ip_address>"
 
 ```
+## Running the `remote-networking-qperf.sh` test
+Run the `remote-networking-qperf.sh` script in host A. The script needs the following inputs to run:
```

Comment:
This is great, but the params are likely to change at some point. To avoid having to update both the script *and* this doc, I'd be tempted to say something like the following, since it's guaranteed that the script will always know which options it needs :)...

> The script requires a number of options to be specified. Run `remote-networking-qperf.sh -h` for details.